openml/__init__.py

```diff
@@ -37,10 +37,11 @@
 from .runs import OpenMLRun
 from . import flows
 from .flows import OpenMLFlow
-from . import setups
 from . import study
-from .study import OpenMLStudy
+from .study import BaseStudy, OpenMLStudy, OpenMLBenchmarkSuite
```

Comment:
Please do not make `BaseStudy` available to the user(s).

Reply:
Removed.